### ISSUE TITLE
The docs are failing to build

### ISSUE THREAD

**Issue by jeffwidman** [python-zk/kazoo#699]
The docs integration has been broken the last few years. 

I just fixed the GitHub webhook integration, but I also noticed the docs builds are currently broken: https://readthedocs.org/projects/kazoo/builds/

**Comment by a-ungurianu** [comment]
Good catch. Looking at the readthedocs, it seems that we moved requirements to `constraints.txt` and never updated the read the docs steps to follow.

No idea where those are held, but I assume access to readthedocs is needed to fix.

**Comment by ceache** [comment]
We moved the docs building requirements to setup.cfg, e.g. "pip install
kazoo[docs]"

I thought I had tested the docs build but if I broke it, I'll see if I can
fix it today.

The docs build itself that happens on readythedocs and seems to still
depend on python 2.7 (from the warning in the logs)?

It also seems like the readythedocs project maintainers are not sync'up
with GitHub. @Jeff Widman ***@***.***> could you have a look and
tell me what the build instructions over there?

**Comment by jeffwidman** [comment]
@ceache I'd invited you when we got upgraded to admins a few weeks back. I just checked and your invitation had expired, so I just re-invited you... that should give you full admin rights to fixup whatever is needed here.

@a-ungurianu I opted to only upgrade @StephenSorriaux / @ceache when I myself was added as an admin a few months back because the three of us have been around for a couple of years, and I hadn't seen much of you for several months. Although that's changed lately, which is awesome! Stick around a bit longer and be more than happy to move you to admin as well. Hope that makes sense.
